File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/typemap_namespace_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'typemap_namespace'

include Typemap_namespace

raise RuntimeError if test1("hello") != "hello"

raise RuntimeError if test2("hello") != "hello"

File: support_tools/swigwin-3.0.12/Examples/test-suite/csharp/typemap_namespace_runme.cs
// This test tests all the methods in the C# collection wrapper

using System;
using typemap_namespaceNamespace;

public class typemap_namespace_runme {

  public static void Main() {
    if (typemap_namespace.test1("hello") != "hello")
      throw new Exception("test1 failed");
    if (typemap_namespace.test2("hello") != "hello")
      throw new Exception("test2 failed");
  }

}

